src/config/xss.ts

```diff
@@ -0,0 +1,7 @@
+import { FilterXSS } from "xss";
+
+export default new FilterXSS({
+  whiteList: {}, // no tags allowed
```

Comment:
do we need to specify this at all?

Reply:
Yes, if we want all scripts to be cleaned in request.